app/views/allocations/_allocate_primary_pom.html.erb

```diff
@@ -5,7 +5,15 @@
     - <%= mail_to(@pom_emails[allocation.primary_pom_nomis_id]) %>
   <% end %>
   <br/>
-  Tier: <%= allocation.allocated_at_tier %>
+  Tier: <%= allocation.allocated_at_tier %></p>
+  <% if allocation.override_reasons.present? %>
+    <p class="govuk-body govuk-!-margin-bottom-1">
+      <%= display_override_pom(allocation) %></p>
+    <p class="govuk-body govuk-!-margin-bottom-1">Reason(s):</p>
+    <% JSON.parse(allocation.override_reasons).each do | reason| %>
```

Comment:
This doesn't look right - none of the other code needs to JSON.parse override_reasons. I think it might be YAML serialised...?